sqlserver/tests/test_integration.py

```diff
@@ -38,7 +38,7 @@ def test_check_invalid_password(aggregator, init_config, instance_docker):
 
 def test_check_docker(aggregator, init_config, instance_docker):
     sqlserver_check = SQLServer(CHECK_NAME, init_config, [instance_docker])
-    sqlserver_check.check(instance_docker)
+    sqlserver_check.run()
```

Comment:
So we propagate errors, let's use https://datadoghq.dev/integrations-core/ddev/plugins/#check-execution